### DISCUSSION TITLE
Does this work on Raspberry Pi Ubuntu?

### DISCUSSION THREAD

**Discussion by Wozza2014** [gogglesmm/gogglesmm#103] [Q&A]
Hi.

Does this work on Raspberry Pi Ubuntu? I added the ppa, updated and tried to install but I got 'E: unable to locate package'.

Regards.

**Comment by gogglesguy** [comment]
Currently I only build the AMD64 and i386 targets. What's your hardware platform for Raspberry Pi Ubuntu?

1. ARM ARMv8 (arm64)
2. ARM ARMv7 Soft Float (armel)
3. ARM ARMv7 Hard Float (armhf)

**Comment by gogglesguy** [comment]
Looks like most of the arm builds are building fine. So wait a few hours and try again. Let me know how it runs.

**Reply by Wozza2014** [reply]
Hi.

I think my pi is Armv8 (arm64) - it's the pi 400.

Cool. I'll give it a few hours and try again. Do I just add the PPA and apt install like before?

Many thanks.

**Reply by gogglesguy** [reply]
I think so.

**Comment by Wozza2014** [comment]
Hi.

Thanks for compiling it for arm. 

It installed fine and runs well. The only issue I have is the artist list doesn't seem to be working. I have 950 albums showing in the album list but only 35 artists showing in the artist list, even though I have 100's of artists.  I can still search and find the album I want. All my files are Opus.

Regards.

**Reply by gogglesguy** [reply]
Do you have a lot of compilation albums by any chance? Technically the list of artists shown are the Album Artists. When you right-click 'edit' any specific track it should show you which 'Album Artist' it is grouped under.

**Reply by gogglesguy** [reply]
Alternatively, there's an import option called 'detect compilations by folders'  which if turned on may have grouped artists together.

**Reply by Wozza2014** [reply]
Hi.

No I hardly have any compilations. I'm strictly an album man - I hate comps hehe.

I'll have another look and get back to you.

Regards.
